patches/api/0417-Add-mob-spawner-ranges-API.patch

```diff
@@ -0,0 +1,71 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: DavidTs93 <david.ts93@gmail.com>
+Date: Fri, 7 Apr 2023 19:28:25 +0300
+Subject: [PATCH] Add mob spawner ranges API
+
+
+diff --git a/src/main/java/org/bukkit/block/CreatureSpawner.java b/src/main/java/org/bukkit/block/CreatureSpawner.java
+index 5bbae759ce39d42886994e500fd9454ec328f804..1e79a6068297fc96d5eca215a802a1345b70aa5a 100644
+--- a/src/main/java/org/bukkit/block/CreatureSpawner.java
++++ b/src/main/java/org/bukkit/block/CreatureSpawner.java
+@@ -174,33 +174,52 @@ public interface CreatureSpawner extends TileState {
+      * in order for this spawner to be active.
+      */
+     public void setRequiredPlayerRange(int requiredPlayerRange);
+-
++    
++    // Paper start
+     /**
+-     * Get the radius around which the spawner will attempt to spawn mobs in.
++     * Get the horizontal radius around which the spawner will attempt to spawn mobs in.
+      * <br>
+      * This area is square, includes the block the spawner is in, and is
+      * centered on the spawner's x,z coordinates - not the spawner itself.
+      * <br>
+-     * It is 2 blocks high, centered on the spawner's y-coordinate (its bottom);
++     * It is {@link #getSpawnRangeVertical()} + 1 blocks high, centered on the spawner's y coordinate (its bottom);
+      * thus allowing mobs to spawn as high as its top surface and as low
+-     * as 1 block below its bottom surface.
++     * as {@link #getSpawnRangeVertical()} blocks below its bottom surface.
+      * <br>
+      * Default value is 4.
+      *
+-     * @return the spawn range
++     * @return the horizontal spawn range
+      */
+     public int getSpawnRange();
+ 
+     /**
+-     * Set the new spawn range.
++     * Set the new horizontal spawn range.
+      * <br>
+      *
+-     * @param spawnRange the new spawn range
++     * @param spawnRange the new horizontal spawn range
+      * @see #getSpawnRange()
+      */
+     public void setSpawnRange(int spawnRange);
+ 
+-    // Paper start
```

Comment:
I would keep the comment here. You don't need to include Javadoc changes inside paper start/end comments.